app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::Base
+  before_action :set_current_user
+
+  private
```

Comment:
- [ ] Great Job working on this milestone. You need to have a `current_user` method similar to below and I don't think it has to be `private` because this method is accessed by the entire application.
```
def current_user
  User.first
end
```